recipe/meta.yaml

```diff
@@ -12,7 +12,7 @@ source:
 
 build:
   skip: true  # [not linux]
-  number: 0
+  number: 1
```

Comment:
FWIW, since this keeps the same pin, just moves where it comes from, it doesn't _technically_ need a new build. No harm, though.